ci/kokoro/docker/e2e/spanner-e2e.cc

```diff
@@ -0,0 +1,28 @@
+#include <grpc++/grpc++.h>
+#include "google/spanner/admin/instance/v1/instance_admin.gapic.h"
+
+using ::google::spanner::admin::instance::v1::ListInstanceConfigsRequest;
+
+int main() {
+  auto creds = grpc::GoogleDefaultCredentials();
+  auto stub = CreateInstanceAdminStub(creds);
+  InstanceAdmin client(std::move(stub));
+  ListInstanceConfigsRequest request;
+  if (const char* project_id = std::getenv("TEST_PROJECT_ID")) {
```

Comment:
Why not use `GOOGLE_CLOUD_PROJECT` as the environment variable, I think almost all examples and tools are standardizing on it.

Reply:
Done